app.rb

```diff
@@ -16,7 +17,7 @@ class MakersBnB < Sinatra::Base
   end
 
   post "/spaces" do
```

Comment:
should we have a 'get 'spaces/new' route that renders the form? and a post 'spaces' route from the new.erb form?